src/editorKit.ts

```diff
@@ -75,7 +77,8 @@ export default class EditorKitBase {
   private fileSafeClass?: any
   private fileSafeInstance?: any
 
-  private note: any
+  private currentNote?: ItemMessagePayload
+  private previousNote?: ItemMessagePayload
```

Comment:
Hmm this naming is confusing. Here we're not saying that this value holds the previously selected note with a different UUID, but instead holding the previous content value that we have received, which could be for the same note as the currentNote, correct? 

Also, it doesn't seem to be necessary. You can hang on to the previous note inline before setting this.currentNote.

Reply:
Right 👍🏽